nifi-nar-bundles/nifi-grpc-bundle/nifi-grpc-common/pom.xml

```diff
@@ -44,6 +44,11 @@
             <artifactId>nifi-security-utils-api</artifactId>
             <version>2.0.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-security-utils</artifactId>
+            <version>2.0.0-SNAPSHOT</version>
+        </dependency>
```

Comment:
Unfortunately the `nifi-security-utils` library has a larger set of transitive dependencies than it should right now. This is a longer-term goal, but for the moment, I recommend avoiding it as a runtime dependency.